extension/src/experiments/index.ts

```diff
@@ -517,6 +522,7 @@ export class Experiments extends BaseRepository<TableData> {
 
   private notifyColumnsChanged() {
     this.columnsChanged.fire()
+    setContextValue('dvc.project.hasData', this.getHasData())
```

Comment:
This should copy the pattern we use for `hasCheckpoints` in `WorkspaceExperiments`:

```
      onDidChangeCheckpoints(() => {
        const workspaceHasCheckpoints = Object.values(this.repositories).some(
          experiments => experiments.hasCheckpoints()
        )

        setContextValue('dvc.experiment.checkpoints', workspaceHasCheckpoints)
      })
```

Reply:
As in using a variable for the second parameter? Or do you mean that the whole thing is not where it should be?

Reply:
I figured it out, will be in the follow up